packages/vuetify/src/mixins/menuable.ts

```diff
@@ -1,12 +1,16 @@
-import Vue from 'vue'
-
+// Mixins
 import Positionable from './positionable'
-
 import Stackable from './stackable'
+
+// Utilities
 import { consoleError } from '../util/console'
+import mixins, { ExtractVue } from '../util/mixins'
+
+// Types
+import Vue, { VNode } from 'vue'
 
 /* eslint-disable object-property-newline */
-const dimensions = {
+const dimensions: any = {
```

Comment:
not really `any`, it has precisely described structure, i think that the type should be imferred from the oject structure, didn't check it though